test/fixtures/typescript/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": "src",
```

Comment:
It would be great to have a similar test for `jsconfig.json`.

Reply:
@iansu running the end to end test got rid of some files, added them back again!